bridge/bridge/NetworkUtils.py

```diff
@@ -68,6 +71,15 @@ async def send_transaction(self, transaction):
 
 		transaction_hash = facade.hash_transaction(transaction)
 		await connector.announce_transaction(transaction)
+
+		is_unconfirmed = await connector.try_wait_for_announced_transaction(
+			transaction_hash,
+			TransactionStatus.UNCONFIRMED,
```

Comment:
Not sure if the transaction status here makes sense. There might be a race where the transaction is announced and gets confirmed before the status is checked.  As with the Symbol connector, the confirmed status is also checked.

Reply:
it can wait for either UNCONFIRMED or CONFIRMED, so i think it makes sense? unless i misunderstand your point?